src/event.rs

```diff
@@ -40,6 +40,7 @@ pub struct ActionContext<'a, N: 'a> {
     pub received_count: &'a mut usize,
     pub suppress_chars: &'a mut bool,
     pub last_modifiers: &'a mut ModifiersState,
+    pub _scrolled_lines: f32,
```

Comment:
In Rust, leading member names with a `_` indicates that the value is unused. If you want to indicate visibility, that's done with the `pub` qualifier (or lack thereof).